lib/diagnostic.rb

```diff
@@ -38,31 +39,31 @@
 
 # define your method here
```

Comment:
This works, but we were looking for a new method defined conventionally for a predicate, such as:

```
def num_odd? (n)
  # something with n
end
```